src/poker_now_log_converter/hand.py

```diff
@@ -250,11 +250,16 @@ def format_as_pokerstars_hand(self, currency: str, currency_symbol: str, timezon
         date_formatted = self.hand_start_datetime.strftime(f"%Y/%m/%d %H:%M:%S {timezone}")
 
         if not self.dealer:
-            # Dead button, so select the seat two places before the big blind to be the button
+            # Dead button, so select the seat three places before the first available seat to be the button
             # This may be an empty seat
-            # The small blind may also be empty
             # Seat numbers are 1-indexed
-            button_seat_id = ((self.big_blind_seats[0].seat_number - 3) % 10) + 1
+            available_seats = [seat for seat in self.seats if seat.stack_size > 0]
+            if available_seats:
+                # first_available_seat_number = available_seats[0].seat_number
```

Comment:
Did you end up opting not to calculate the button based from `first_available_seat_number`? The comment above still refers to `first_available_seat_number`.